services/dynamic/dynamic-helpers.js

```diff
@@ -20,6 +12,14 @@ const queryParamSchema = Joi.object({
   .rename('uri', 'url', { ignoreUndefined: true, override: true })
   .required()
 
+function createRoute(which) {
+  return {
+    base: `badge/dynamic/${which}`,
+    pattern: '',
+    queryParamSchema,
+  }
+}
+
 module.exports = {
   createRoute,
   queryParamSchema,
```

Comment:
With the updates to `createRoute`, does the `queryParamSchema` still need to be exported?

Reply:
Nope! Updated.